### PR TITLE
fix more grammar mistakes

### PR DESCRIPTION
**Pull request by maksim-charchuk**
There are 2 more ones

### CODE REVIEW

File: tips/21-08-2017/Readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-We have [discussed][1] earlier how to significally reduce a bundle size which imports `lodash` inside. Now we will learn more (in 2 parts) about specific webpack options and settings used in order to reduce the bundle still not breaking the functionality too much.
+We have [discussed][1] earlier about how to significantly reduce the size of a bundle that imports `lodash` inside itself. Now we will learn more (in 2 parts) about specific webpack options and settings used in order to reduce the bundle still not breaking the functionality too much.
```
Comment (jakwuh) [inline tips/21-08-2017/Readme.md:1]: `about` seems to be redundant here

Comment (maksim-charchuk) [reply tips/21-08-2017/Readme.md:1]: It's been verified by an American

Comment (jakwuh) [reply tips/21-08-2017/Readme.md:1]: Internet says me this is not correct:
https://www.englishforums.com/English/DiscussVsDiscussAbout/zlpwz/post.htm
https://english.stackexchange.com/questions/146833/is-discuss-about-grammatically-incorrect

Comment (maksim-charchuk) [reply tips/21-08-2017/Readme.md:1]: Well, it might have a slightly different meaning. Let's skip it for a while

Comment (jakwuh) [reply tips/21-08-2017/Readme.md:1]: May you update your PR?

**Pull request closed without merging**